library/src/main/java/com/datatorrent/lib/io/jms/AbstractJMSInputOperator.java

```diff
@@ -514,6 +514,11 @@ public WindowDataManager getWindowDataManager()
     return this.windowDataManager;
   }
 
+  public void setTransacted(boolean value) 
```

Comment:
Please remove whitespace at end of line (they tend to cause regex searches to sometimes fail on such lines).